schema/home/home_page_hero_units.js

```diff
@@ -21,10 +22,34 @@ const HomePageHeroUnitType = new GraphQLObjectType({
       type: GraphQLString,
       resolve: ({ link }) => link,
     },
+    description: {
+      type: GraphQLString
+    },
     title: {
       type: GraphQLString,
       resolve: ({ mobile_title }) => mobile_title,
     },
+    title_image_url: {
+      type: GraphQLString,
+    },
+    type: {
+      type: GraphQLString,
+    },
+    name: {
+      type: GraphQLString,
+    },
+    menu_color_class: {
```

Comment:
I wonder if we should clean up this data model vs. passing Gravity values as-is. For instance, we could consolidate some of these fields to be like...

```
{
  mode: enum {
    LEFT_DARK
    LEFT_LIGHT
    CENTERED_LIGHT
    CENTERED_DARK
    RIGHT_DARK
    RIGHT_LIGHT
  }
}
```

vs. how we combine `menu_color_class` `link_color_hover_hex` and `type` now to determine what it looks like.